### PR TITLE
Feature/match any word

### PR DESCRIPTION
**Pull request by kmachado**
need a special review on the description of the functions

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
-﻿# 1.8.0-1.8.3
+# 1.9.0
+- `makeRegex` curry implementation of RegExp construction.
```
Comment (frozegnome) [inline CHANGELOG.md]: Change 'curry' to 'curried'

File: README.md [hunk 2]
```diff
@@ -147,6 +147,17 @@ Example: `{ a: { b: { c: 1 } } } => { 'a.b.c' : 1 }`.
 ### testRegex
 `regex -> string -> bool` Just like rambda test, creates a function to test a regex on a string.
 
+### makeRegex
+`makeRegex -> options -> string -> regex` Just a curry implementation of RegExp construction.
```
Comment (frozegnome) [inline README.md]: Change 'curry' to 'curried'

File: README.md [hunk 3]
```diff
@@ -147,6 +147,17 @@ Example: `{ a: { b: { c: 1 } } } => { 'a.b.c' : 1 }`.
...
+
+### makeAndTest
+`makeAndTest -> options -> (string) -> ` makes and tests a RegExp with makeRegex and testRegex
+
+### matchAnyWord
+`matchAnyWord -> string -> string -> bool` 
+takes a string to split into words and returns true if second string match any of the words
```
Comment (frozegnome) [inline README.md]: change 'if second string...' to 'if the second string'

File: README.md (hunk 2 above)
Comment (sadasant) [inline README.md]: And `of RegExp construction` to `of the RegExp construction`.

**Comment by daedalus28** [comment]
closing in favor of #66

**Pull request closed without merging**